Changelog.md

```diff
@@ -7,6 +7,13 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 122 - 2023-11-02
+
+### Fixed
+
+-   `JLink` warning when using untested JLink version as expected by nrfutil
+    device cli
```

Comment:
I'm not really sure what this means.

What about 
```suggestion
-   Check installed JLink version against expected version given by nrfutil instead of the version bundled by launcher.
```